block/manager.go

```diff
@@ -144,24 +143,27 @@ func (m *Manager) Start(ctx context.Context) error {
 		}
 	}
 
+	// Fullnode loop can start before syncing from DA
 	if !isSequencer {
 		go uevent.MustSubscribe(ctx, m.Pubsub, "applyGossipedBlocksLoop", p2p.EventQueryNewNewGossipedBlock, m.onNewGossipedBlock, m.logger)
+		go m.RetrieveLoop(ctx)
```

Comment:
I think there's issue here.
now `syncUntilTarget` can be called in parallel, which makes no sense.
once from `syncBlockManager` and 2nd from `RetrieveLoop`

Reply:
true. the retrieve loop should start only when the synblockmanager has finished, otherwise there can be two syncing processes in parallel. i fixed this.

Reply:
i also moved that sequencers just wait for the da to be synced only to start producing blocks, since blobs can be retrieved even if da is not fully synced. i also renamed the channel to be more clear.